linkedin/linkedin.lua

```diff
@@ -0,0 +1,66 @@
+--[[
+# [LinkedIn API](https://developer.linkedin.com/docs/ref/v2/profile/basic-profile) connector.
+
+### Authorization
+
+Authorization use [OAuth 2.0](https://developer.linkedin.com/docs/rest-api).
```

Comment:
This is the link for authorization explanation
https://developer.linkedin.com/docs/oauth2